src/components/Settings/Account.js

```diff
@@ -22,7 +22,15 @@ class Account extends Component {
   onChange = e => {
     this.setState({ [e.target.name]: e.target.value });
   };
-
+  onSubmit = ()=>{
+    const name = this.state.username;
+    if(name !== this.props.authUser.name){
+      const {id,email,isVendor,isAdmin} = this.props.authUser;
+      console.log(id,email,isVendor,isAdmin,name);
+      const updatedUser = {name,email,isVendor,isAdmin};
```

Comment:
Just have name in this. No need of email, isVendor and isAdmin.

Reply:
Cannot remove this